output/build.sh

```
@@ -0,0 +1,17 @@
+# 팀 레포에서 개인 레포로 코드 복사
+
+#!/bin/sh
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)